docs/en/Running-Alluxio-Locally.md

```diff
@@ -34,13 +34,23 @@ repeatedly input the password, you can add the public ssh key for the host into
 $ ./bin/alluxio format
 ```
 
-# Start Alluxio Filesystem Locally
+# Step 1: Start Alluxio Filesystem Locally
 
-## Start Alluxio with sudo privileges
-
-By default, on startup Alluxio will create a
+By default, Alluxio Filesystem will create and use
 [RAMFS](https://www.kernel.org/doc/Documentation/filesystems/ramfs-rootfs-initramfs.txt) as its in-memory data storage.
-This step requires sudo privileges to perform "mount", "umount", "mkdir" and "chmod" operations. There are two approaches to achieve this:
+Depends on the operating systems and environment, it may require different privileges to complete this step.
+
+## Running on MacOS
```

Comment:
as discussed offline, let's reorganize this to shift the focus from `sudo` onto how to start Alluxio in the common case (where it just works)